tests/index.js

```diff
@@ -106,6 +106,8 @@ describe('Thorn', () => {
                         name: 'TestRecord1',
                         testField1: 'TestField1data',
                         testField2: 'TestField2data',
+                        testField3: false,
```

Comment:
Imho the test does make sense, I'm worried that it might be highlighting a bigger problem than the one I was initially expecting and that now seems to be similar to the one we had on [wrapRequest](https://github.com/sugarcrm/thorn/pull/79/files#r96758868) that has been fixed as part of #79.